frontend/src/App.js

```diff
@@ -13,14 +13,10 @@ import Menu from './components/menu'
 // local imports
 import './static/css/global-styles.scss'
 import routers from './routers'
+import notFound from './containers/404/not-found'
```

Comment:
You can create a folder "errors" instead of "404", because we can create later a new screen to treat others erros